src/recorder.js

```diff
@@ -0,0 +1,17 @@
+export default class Recorder {
```

Comment:
Please add a good comment for `Recorder` and `record`, as this is the most important public concept in `basictracer`.